cheatsheets/Threat_Modeling_Cheat_Sheet.md

```diff
@@ -17,25 +17,23 @@ Following the guidance in this cheat sheet, the assessors will list all possible
 
 Understand the Relation between Risk, Threats, and Vulnerabilities.
 
-## Understand Threat Modeling Terminologies
+## Threat Modeling Terminologies
 
-### Asset, Threat Agent, Attack Surface, Likelihood, Impact, Control, Mitigation, Tractability Matrix
+*Information Asset*, a body of knowledge that is organized and managed as a single entity. Like any other corporate asset, an organization's information assets have financial value.
 
-Information Asset: is a body of knowledge that is organized and managed as a single entity. Like any other corporate asset, an organization's information assets have financial value.
+*Threat Agent*, an individual or group that can manifest a threat. It is fundamental to identify who would want to exploit the assets of a company, and how they might use them against the company. 
 
-Threat Agent: The term Threat Agent is used to indicate an individual or group that can manifest a threat. It is fundamental to identify who would want to exploit the assets of a company, and how they might use them against the company. 
+*Attack Surface*, the sum of the different points (the "attack vectors") where an unauthorized user (the "attacker") can try to enter data to or extract data from an environment.
 
-Attack Surface: The attack surface of a software environment is the sum of the different points (the "attack vectors") where an unauthorized user (the "attacker") can try to enter data to or extract data from an environment.
+*Likelihood*, the possibility of a athreat event occuring where a threat actor will exploit a weakness. The likelihood of threat events resulting in adverse impacts estimates the possibility that a threat event would result in an actual outcome. The combined analysis of both threat assessment vectors impacts established an overall threat likelihood.
```

Comment:
Small typo here, to the word **threat** you added **a**:
 - *Likelihood*, the possibility of a **a**threat

Reply:
@mackowski good catch. Fixed.